queries/buildings.jinja2

```diff
@@ -73,6 +79,8 @@ SELECT
     NULL AS roof_orientation,
     "addr:housenumber" AS addr_housenumber,
     "addr:street" AS addr_street,
+    tags->'addr:floor' AS addr_floor,
+    tags->'level' as level,
```

Comment:
Let's drop the add_floor and level bits from this until there's more of a use case.